mclocations/minecraft.py

```diff
@@ -0,0 +1,89 @@
+import re
+
+from mcrcon import MCRcon
+
+from mclocations.models import Player, Position
+
+
+class MinecraftServer:
+    def __init__(self, host, password, port=25575):
+        self.mcr = MCRcon(host, password, port)
+        self.mcr.connect()
+
+    def disconnect(self):
+        self.mcr.disconnect()
+
+    def get_players(self):

```

Comment:
Yeah, this too needs more reading. Why does self need to be passed into this method? I take it python doesn't have a `this` context?

Reply:
There is no implicit this. The first argument to an instance method is the instance. 

If you want a deep dive, check out http://neopythonic.blogspot.com/2008/10/why-explicit-self-has-to-stay.html where a proposal to make `self` implicit instead is discussed by Guido van Rossum.

If you want to hurt peoples brain, rename `self` something else. I'm pretty sure you can put anything there, but I've honestly never seen anything used but `self`.